internal/auth0/role/resource_test.go

```diff
@@ -71,163 +140,3 @@ func TestAccRole(t *testing.T) {
 		},
 	})
 }
-
-const testAccRoleEmpty = `
-resource auth0_role the_one {
-  name = "The One - Acceptance Test - {{.testName}}"
-}
-`
-
-const testAccRoleAux = `
-resource auth0_resource_server matrix {
-    name = "Role - Acceptance Test - {{.testName}}"
-    identifier = "https://{{.testName}}.matrix.com/"
-    scopes {
-        value = "stop:bullets"
-        description = "Stop bullets"
-    }
-    scopes {
-        value = "bring:peace"
-        description = "Bring peace"
-    }
-}`
-
-const testAccRoleCreate = testAccRoleAux + `
-resource auth0_role the_one {
-  name = "The One - Acceptance Test - {{.testName}}"
-  description = "The One - Acceptance Test"
-  permissions {
-    name = "stop:bullets"
-    resource_server_identifier = auth0_resource_server.matrix.identifier
-  }
-}
-`
-
-const testAccRoleUpdate = testAccRoleAux + `
-resource auth0_role the_one {
-  name = "The One - Acceptance Test - {{.testName}}"
-  description = "The One who will bring peace - Acceptance Test"
-  permissions {
-    name = "stop:bullets"
-    resource_server_identifier = auth0_resource_server.matrix.identifier
-  }
-  permissions {
-    name = "bring:peace"
-    resource_server_identifier = auth0_resource_server.matrix.identifier
-  }
-}
-`
-
-const testAccRoleEmptyAgain = `
-resource auth0_role the_one {
-  name = "The One - Acceptance Test - {{.testName}}"
-  description = " "
-}
-`
-
-func TestAccRoleResourcePermissions(t *testing.T) {
```

Comment:
This test isn't needed any more as permissions won't be manageable through the role resource directly but instead through the 1:1 and 1:many resources.